website/pages/chapters/ridge.js

```diff
@@ -0,0 +1,845 @@
+          ! We try out lots of possible settings of <IM math={`\\lambda`} /> and
+          see which one does best on a validation set or in cross validation.
+        </p>
+      </section>
+
+      <section>
+        <h2>Practical considerations</h2>
+
+        <p>
+          By this point, we have hopefully convinced you that regularization can
+          be a powerful tool in preventing overfitting. While everything we have
+          said above works mostly well in theory, there are some very real
+          practical considerations we can make to improve this process and make
+          sure it works well.
+        </p>
+
+        <p>
+          The first is a rather subtle point about which coefficients you should
+          regularize. In our explanation above, we showed that you should
+          regularize the whole coefficient vector{" "}
+          <IM math={`w = [w_0, w_1, ..., w_D]`} />. That might have been a bit
+          overkill and will likely cause problems in practice{" "}
+          <MarginNote>
+            <img
+              className="border-image"
+              src="/animations/ridge/all_the_coeffs.png"
+              alt="Meme"
+            />
+            Editor's note: I think I made this meme, but there is a non-zero
+            chance I don't remember finding it on the internet.
+          </MarginNote>
+          .
+        </p>
+
+        <p>
+          For <em>most</em> of the features, it makes sense to regularize their
+          coefficients to stop overfitting. The one place that doesn't make
+          sense is the <em>intercept</em> <IM math={`w_0`} />. For the housing
+          price example, the <IM math={`y`} /> values can be quite large. We
+          don't want to penalize the model for having to have a large
+          coefficient just to get up to the house prices that are in the
+          thousands at a minimum.
+        </p>
+
+        <p>
+          There are two very common ways to deal with this problem so we don't
+          regularize the intercept:
+        </p>
+        <ul>
+          <li>
+            Change the quality metric to not include the intercept in the
+            regularization term. Define{" "}
+            <IM math={`w_{rest} = [w_1, ..., w_D]`} /> to not include the
+            intercept <IM math={`w_0`} />.
+            <BM
+              math={`\\hat{w_0}, \\hat{w}_{rest} = \\min_{w_0, w_{rest}} RSS(w_0, w_{rest}) + \\lambda \\left|\\left|w_{rest} \\right| \\right|_2^2`}
+            />
+            Our predictor will then be in terms of these values.
+            <BM
+              math={`\\hat{y} = \\hat{w_0} + \\sum_{j=1}^D \\hat{w}_{rest,\\ j} \\cdot x`}
+            />
+          </li>
+          <li>
+            Center the data so the <IM math={`y`} /> values are have mean 0. If
+            this is the case, then favoring a <IM math={`w_0`} /> is not a
+            problem since the data is centered at 0.
+          </li>
+        </ul>
+
+        <p>
```

Comment:
I'd add an `<h3>` tag called "Normalization" as that's a pretty important practical detail.

Reply:
And in the previous bullet id introduce the term normalization, so it's crystal clear you're talking about normalization in the previous section.

Reply:
Hmmm, technically you don't need to normalize the data for the intercept problem. I've always thought of normalization as mean centering and scaling. You don't need to change the scale of the `ys` for the intercept problem, just make sure they are mean 0.